src/amp-story-player-impl.js

```diff
@@ -66,17 +66,19 @@ export const IFRAME_IDX = '__AMP_IFRAME_IDX__';
  * Note that this is a vanilla JavaScript class and should not depend on AMP
  * services, as v0.js is not expected to be loaded in this context.
  */
-export class AmpStoryPlayer {
+export class AmpStoryPlayer extends HTMLElement {
```

Comment:
I believe they are public by default. 

@jridgewell is there a way for us to not expose certain methods in a custom element?

Reply:
What methods do you want to be private? The general convention is to prefix `_` to the method/property name. You can get more complicated by using a Map to store the data, but that's a lot slower.

Reply:
> What methods do you want to be private?

We'd like to make the current methods that contain a `_` suffix private. 

> The general convention is to prefix _ to the method/property name.

It seems that by adding the prefix I can still access them. By e.g.:
```
player = body.querySelector('amp-story-player');
player._privateMethod(); // Still accessible.
```

 @newmuis @gmajoulet do you think this would be a problem?

Reply:
JS doesn't yet have private methods. You can use a `Map` to store data in the map and _not_ on the element. But it is slow and will have memory leaks. Just store private methods on the element with a prefix.

Reply:
this SGTM

Reply:
I changed the private methods and properties to use the prefix as Justin suggested, wasn't sure if changing the properties was necessary, but thought it might be good to have consistency across the file.